src/main/java/au/org/emii/ncdfgenerator/NcdfEncoder.java

```diff
@@ -126,8 +124,7 @@ public InputStream get() throws Exception
 					" order by " + orderClause +
 					";" ;
 
-				// TODO logger
-				System.out.println( "instanceId is " + instanceId + " " + query);
+				logger.info( "instanceId " + instanceId + ", " + query );
```

Comment:
Might be a little chatty for the prod web service logs.  Perhaps use debug and change resources/log4j.properties to report debug level during testing while your still working on this